src/connection/sql_connection.go

```diff
@@ -70,6 +73,14 @@ func CreateConnectionURL(args *args.ArgumentList) string {
 	query.Add("dial timeout", args.Timeout)
 	query.Add("connection timeout", args.Timeout)
 
+	mockURL, err := url.Parse("http://example.com?" + args.ExtraConnectionURLArgs)
```

Comment:
I mean the mockURL thing, I think we can remove it 
```suggestion
	if args.ExtraConnectionURLArgs != "" {
		extraArgsMap, err := url.ParseQuery(args.ExtraConnectionURLArgs)
		if err != nil {
			log.Warn("Unable to process ExtraConnectionURLArgs: %s", err.Error())
		}
		for k, v := range extraArgsMap {
			query.Add(k, v[0])
		}
	}
```